Snippets/Snippets_6/Transports/Throughput/InitialThroughput.config.xml

```diff
@@ -1,10 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
   <!-- startcode TuningFromAppConfig-->
-  <configSections>
-    <section name="TransportConfig"
-             type="NServiceBus.Config.TransportConfig, NServiceBus.Core"/>
-  </configSections>
-  <TransportConfig MaximumConcurrencyLevel="5"/>
+  <appSettings>
+    <add key="MaxConcurrency" value="5"/>
```

Comment:
nowhere yet. I can't update the docs yet since the EndpointConfig renamings aren't in yet. Also not sure how we want to pull this off here since we need to show the code api accessing this setting.